embedded-nal-async/README.md

```diff
@@ -0,0 +1,57 @@
+<!--
+[![crates.io](https://img.shields.io/crates/d/embedded-hal-async.svg)](https://crates.io/crates/embedded-hal-async)
+[![crates.io](https://img.shields.io/crates/v/embedded-hal-async.svg)](https://crates.io/crates/embedded-hal-async)
+[![Documentation](https://docs.rs/embedded-hal-async/badge.svg)](https://docs.rs/embedded-hal-async)
+![Minimum Supported Rust Version](https://img.shields.io/badge/rustc-1.46+-blue.svg)
+-->
+
+# `embedded-hal-async`
+
+An asynchronous Hardware Abstraction Layer (HAL) for embedded systems.
+
+This crate contains asynchronous versions of the [`embedded-hal`] traits and shares its scope and [design goals].
+The purpose of this crate is to iterate over these trait versions before integrating them into [`embedded-hal`].
+
+**NOTE** These traits are still experimental. At least one breaking change to this crate is expected in the future (changing from GATs to `async fn`), but there might be more.
+
+<!-- This project is developed and maintained by the [HAL team][team]. -->
```

Comment:
There's a lot of boilerplate about maintenance and various aspects that need to be updated, as they look like copy-pasta from the e-h crate.